gravitee-policy-ratelimit/src/test/java/io/gravitee/policy/ratelimit/configuration/RateLimitPolicyConfigurationTest.java

```diff
@@ -55,6 +56,8 @@ public void test_quota02() throws IOException {
         Assert.assertTrue(configuration.isAsync());
     }
 
+    // TODO test on old format (no templatableLimit)
```

Comment:
We should probably remove this one :innocent:

Reply:
Done in latest commit